Jellyfin.Api/Controllers/ConfigurationController.cs

```diff
@@ -119,6 +120,51 @@ public ActionResult<MetadataOptions> GetDefaultMetadataOptions()
         return new MetadataOptions();
     }
 
+    /// <summary>
+    /// Gets branding configuration.
+    /// </summary>
+    /// <response code="200">Branding configuration returned.</response>
+    /// <returns>Branding configuration.</returns>
+    [HttpGet("Configuration/Branding")]
+    [ProducesResponseType(StatusCodes.Status200OK)]
+    public ActionResult<BrandOptionsDto> GetBrandingConfiguration()
```

Comment:
There already is a getBrandingOptions in BrandingController. That one needs to be updated to use the DTO and this operation removed.

Reply:
yeah wanted to do that. shouldnt break the api now👀